diff --git a/wasmCompile.mjs b/wasmCompile.mjs
index f9cbb214..ed127489 100644
--- a/wasmCompile.mjs
+++ b/wasmCompile.mjs
@@ -8,7 +8,7 @@
 import { exec } from "child_process";
 import fs from "fs";
 
-const EMSCRIPTEN_VERSION = "4.0.12";
+const EMSCRIPTEN_VERSION = "";
 
 try {
   await new Promise(async (resolve, reject) => {
@@ -50,8 +50,9 @@ try {
       inPath = "'src/hub/dataSources/wpilog/indexer/wpilogIndexer.c'";
       outPath = "'bundles/wpilogIndexer.js' ";
     }
+    var cache = process.env.EMSCRIPTENCACHE;
     exec(
-      `emcc ${inPath} -o ${outPath} -sEXPORT_ES6 -sEXPORTED_FUNCTIONS=_run,_malloc -sEXPORTED_RUNTIME_METHODS=HEAPU8,HEAPF64,HEAPU32 -sALLOW_MEMORY_GROWTH -sMAXIMUM_MEMORY=4294967296 -O3`,
+      `emcc ${inPath} -o ${outPath} --cache ${cache} -sEXPORT_ES6 -sEXPORTED_FUNCTIONS=_run,_malloc -sEXPORTED_RUNTIME_METHODS=HEAPU8,HEAPF64,HEAPU32 -sALLOW_MEMORY_GROWTH -sMAXIMUM_MEMORY=4294967296 -O3`,
       (error, stdout, stderr) => {
         console.log(stdout);
         console.error(stderr);
